Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -976,6 +978,10 @@ internal struct LongPathInfo {
                     _analyzer = null;
                 }
 
+                if (_idleNodeModulesTimer != null) {
+                    _idleNodeModulesTimer.Dispose();

```

Comment:
In addition to disposing the timer, we should also unsubscribe to the event handler before disposing (otherwise the callbacks that are already scheduled will continue to fire).